ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -84,6 +86,12 @@ public void setElevation(T view, float elevation) {
     // Do nothing on API < 21
   }
 
+  @ReactProp(name = PROP_Z_INDEX)
+  public void setZIndex(T view, float zIndex) {
```

Comment:
Mmmm all the other numerical props were float, so I just copied those. I'm *guessing* this is because React only has PropTypes.number, which is basically float.

I can totally make a custom integer PropType though. Will need to test how the bridge reacts if you still sent a float through, and the setZIndex param is int.